README.md

```diff
@@ -87,11 +87,12 @@ npm run
 
 ## API endpoints
 
-| Endpoint                       | Description  |
-| :----------------------------- | :----------- |
-| `GET: /health`                 | Health       |
-| `GET: /v1/entities`            | Entities     |
-| `GET: /v1/entities/<entityId>` | Entity by ID |
+| Endpoint                       | Description       |
+| :----------------------------- | :---------------- |
+| `GET: /health`                 | Health            |
+| `GET: /v1/entities`            | Entities          |
+| `GET: /v1/entities/<entityId>` | Entity by ID      |
+| `PATCH: /forms/<id>`           | Update Form by ID |
```

Comment:
I will add other endpoints in another PR. I don't want to hold this PR for too long